articles/frontend-is-not-your-enemy.md

```diff
@@ -0,0 +1,157 @@
+---
+authorName: Janko Sokolović
+authorGithubUsername: jasofalcon
+issue: 81
+title: Frontend Is Not Your Enemy 
+---
+# Frontend Is Not Your Enemy
+
+For many years, developers are discriminating frontend technologies because of various reasons.
+
+*“It is too volatile!”*, some said
+
+*“But I've just learned AngularJS version 1.0 yesterday, today it's already v3.3.1”*, many yelled while hiding the tears...
+
+*“It doesn't work the same in every browser!”*, was heard from distance...
+
+We admit it. It is partially our fault. Us, engineers.
+
+However, a big portion of the problem are the users. It is similar in other industries. Look at the hardware vendors, trying to create just perfect user experience. Now I have to call my Mom every time I get a new washing machine because I don't understand any of the icons on it. Similarly, frontend has exploded in last decade or so, due to so many different (innovative) requirements to create the most user-friendly web app.
```

Comment:
I would rephrase this section, without this comparison to hardware vendors (and mommy :), just stick to the software industry, if possible.